templates/search.html

```diff
@@ -3,98 +3,117 @@
+          {% endif %}
 
-  <section class="p-strip is-deep">
-    <div class="row">
-        {% if snaps %}
-        <div class="col-8">
-        {% for snap in snaps %}
-          <div class="p-media-object">
-              <a href="/{{ snap.package_name }}" class="p-media-object__image">
-              {% if snap.icon_url %}
-                <img src="{{ snap.icon_url }}"  alt="">
-              {% else %}
-                <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg"  alt="">
-              {% endif %}
-              </a>
-              <div class="p-media-object__details">
-                {# FIXME u-no-margin is a workaround for https://github.com/vanilla-framework/vanilla-framework/issues/1879 #}
-                <h3 class="p-media-object__title u-no-margin">
-                  <a href="/{{ snap.package_name }}">{{ snap.title }}</a>
-                </h3>
-                <ul class="p-media-object__meta-list u-no-margin">
-                  <li class="p-media-object__meta-list-item">
-                    <span class="u-off-screen">Publisher: </span>{{ snap.publisher }}
-                    {% if snap.developer_validation and snap.developer_validation == VERIFIED_PUBLISHER %}
-                      <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="{{ snap.package_name }}-tooltip">
-                        <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg" />
-                        <span class="p-tooltip__message u-align--center" role="tooltip" id="{{ snap.package_name }}-tooltip">Verified account</span>
-                      </span>
-                    {% endif %}
-                  </li>
-                </ul>
-                <p class="p-media-object__content">{{ snap.summary }}</p>
-              </div>
+          <div class="row">
+            <h3>{{ category.name }}</h3>
           </div>
-        {% endfor %}
-        </div>
-        {% else %}
-          {% if featured_snaps %}
-            <h2>Featured snaps</h2>
-            <div class="col-12 p-featured-snaps">
-              {% for snap in featured_snaps %}
-                <a class="p-featured-snaps__item p-media-object" href="/{{ snap.package_name }}">
-                  <span class="p-media-object__image">
-                    {% if snap.icon_url %}
-                      <img class="p-heading-icon__img" src="{{ snap.icon_url }}" alt="">
-                    {% else %}
-                      <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" alt="">
-                    {% endif %}
-                  </span>
-                  <span class="p-media-object__details">
-                    <h4>{{ snap.title }}</h4>
-                  </span>
-                </a>
-              {% endfor %}
-            {% endif %}
+          <div data-category="{{ category.slug }}" class="row js-store-category">
+            {# _category-partial.html loaded via JS #}
```

Comment:
It's just one category to show here. We should be able to render it in HTML here easily and not involve JS (apart from running the carousel).

Reply:
For this first version I just wanted to reuse what I had already built, without needing to add extra calls to the search endpoint. If we decide this doesn't work very well long term, we can definitely implement it a better way.